test/Emitter/SapiStreamEmitterTest.php

```diff
@@ -35,10 +35,14 @@
  */
 class SapiStreamEmitterTest extends AbstractEmitterTest
 {
+    /** @var SapiStreamEmitter */
+    protected $emitterIgnoreContentRange;
```

Comment:
private modifier should be fine since this class is not to be extended.

```suggestion
    private $emitterIgnoreContentRange;
```